ee/control/control_test.go

```diff
@@ -410,6 +410,71 @@ func TestControlServicePersistLastFetched(t *testing.T) {
 	}
 }
 
+func Test_knownSubsystem(t *testing.T) {
```

Comment:
This is also tested within the context of `Fetch` in `TestControlServiceFetch_IgnoresUnknownSubsystems`